apps/ledger-live-mobile/src/screens/Swap/Form/Summary/index.tsx

```diff
@@ -203,7 +206,10 @@ export function Summary({ provider, swapTx: { swap, status, transaction } }: Pro
         </Text>
       </Item>
 
-      <Item title={t("transfer.swap2.form.details.label.fees")} onEdit={onEditNetworkFees}>
+      <Item
+        title={t("transfer.swap2.form.details.label.fees")}
+        onEdit={editableFee && onEditNetworkFees}
+      >
```

Comment:
I'm not familiar with this codebase, but IIUC, for families where fees aren't editable, this will leave the component enabled but just do nothing when clicked? If so, wouldn't it be better to disable or hide it instead?

Reply:
This component have the label, the value and the edit button. With the modification if `onEdit` is false, instead a function, edit button is removed.